sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/rest_client.hpp

```diff
@@ -1423,9 +1434,9 @@ namespace Azure { namespace Storage { namespace Blobs {
          */
         std::vector<BlobItem> Items;
         /**
-         * Array of ListBlobsHierarchySegmentResponseBlobPrefixesItem.
+         * Array of BlobName.
          */
-        std::vector<std::string> BlobPrefixes;
+        std::vector<BlobName> BlobPrefixes;
```

Comment:
Are we making sure this change isn't breaking in the public surface area?

Reply:
This is in `_detail` namespace so it's non breaking.